omnibus/files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -49,6 +49,33 @@
 include_recipe 'private-chef::plugin_discovery'
 include_recipe 'private-chef::plugin_config_extensions'
 include_recipe 'private-chef::config'
+# this implies it is the firsttime elasticsearch is being installed internally.
+# do not enable opscode-solr4 and rabbitmq services
+# TODO: (prajakta) delete the runsv files for opscode-solr4 and rabbitmq if they exist
+directory '/opt/opscode/sv/opscode-solr4' do
+  recursive true
+  action :delete
+  only_if { File.exist?('/tmp/first_time_internal_elasticsearch_install') }
+end
+
+directory '/opt/opscode/sv/rabbitmq' do
+  recursive true
+  action :delete
+  only_if { File.exist?('/tmp/first_time_internal_elasticsearch_install') }
+end
+
+directory '/opt/opscode/sv/opscode-expander' do
```

Comment:
We'll also want to clean up the rabbit/solr log services. 

Should we also be deleting the data directory for solr4/rabbitmq? 

Instead of doing this per directory, is it possible to use the runit service resource w/ action :delete and have it do handle all the directories?

Reply:
Looks like there is no action :delete for the runit cookbook, but the recipe now does delete the log dirs as well. Thank you!